Fix for issues # 59 and 60, reading cluster value in the state file

Prativa20 opened the pull request:
1. https://github.com/terraform-providers/terraform-provider-vra7/issues/60
   Replace the default cluster only if cluster is pecified in the config file

2. https://github.com/terraform-providers/terraform-provider-vra7/issues/59
   Modify the logic to retrieve cluster value, not using the Scale Out template

3. Replace the default business group id in the action template only if specified in the config

raghavav commented on README.md:
```diff
@@ -172,6 +172,8 @@ Optional:
 
 * **resource_configuration** - *This is an optional field. If blueprint properties have default values or no mandatory property value is required then you can skip this field from terraform configuration file. This field contains user inputs to catalog services. Value of this field is in key value pair. Key is service.field_name and value is any valid user input to the respective field.*
 
+* **wait_timeout** - *wait_timeout is optional field with default value of 15 mins. wait_timeout defines the time to wait until resource comes up or destroys resource successfully*
```
Please follow the language style that is used in the rest of the document:

"This is an optional field with a default value of 15. It defines the time to wait (in minutes) for a resource operation to complete successfully."

virajindasrao replied:
Done.

Prativa20 committed:
Fix for issues # 59 and 60, reading cluster value in the state file

1. https://github.com/terraform-providers/terraform-provider-vra7/issues/60
   Replace the default cluster only if cluster is pecified in the config file

2. https://github.com/terraform-providers/terraform-provider-vra7/issues/59
   Modify the logic to retrieve cluster value, not using the Scale Out template

3. Replace the default business group id in the action template only if specified in the config

markpeek reviewed (approved):
LGTM!
